src/main/kotlin/racing/ResultView.kt

```diff
@@ -1,13 +1,19 @@
 package racing
 
-fun printResult(carList: List<Car>) {
-    println("실행 결과")
+fun printResult(cars: List<Car>, tryCount: Int) {
+    repeat(tryCount) { index ->
+        printRecord(cars, index)
+        println()
+    }
+}
 
-    carList.forEach {
-        (0 until it.position).forEach {
-            print("-")
-        }
-        println("")
+fun printRecord(cars: List<Car>, index: Int) {
+    cars.forEach { car ->
+        println("${car.name} : " + "-".repeat(car.records[index]))
```

Comment:
굳이 중간에 + 를 넣지 않아도 괜찮아요!
```suggestion
        println("${car.name} : ${"-".repeat(car.records[index])}")
```